apps/trustlab/src/next-seo.config.js

```diff
@@ -0,0 +1,29 @@
+import site from "@/trustlab/utils/site";
+
+const config = {
+  // Disable indexing while in development
+  dangerouslySetAllPagesToNoFollow: true,
+  dangerouslySetAllPagesToNoIndex: true,
+  openGraph: {
+    type: "website",
+    locale: "en_GB",
+    url: site.environmentUrl,
+    site_name: site.name,
+    images: [
+      {
+        url: `${site.environmentUrl}image.jpg`,
+        width: 1600,
+        height: 800,
+        alt: site.name,
+        type: "image/jpeg",
+      },
+    ],
+  },
+  twitter: {
+    handle: "@Code4Africa",
+    site: "@Code4Africa",
```

Comment:
Add an issue to check with TrustLab team if they have their own Twitter/X handle or not.

Reply:
#1148